### ISSUE TITLE
Module: Sound Sprite Sheeting Library

### ISSUE THREAD

**Issue by mikkoh** [Jam3/tech-we-want-to-use#5]
Dev Time: 2 days

Create a library which can take an audio file which contains multiple other audio files. This library would be able to playback and control this audio sprite sheet. It should also return events for progress and completion of sounds.

**Comment by njam3** [comment]
Howler already handles this pretty well, saves us from reinventing the wheel. https://www.npmjs.com/package/howler

**Comment by jmckinnell** [comment]
Yeah, Howler 2.0 is really nice. It's not in npm just yet so you have to ssh in the package.
https://github.com/goldfire/howler.js/tree/2.0
